doc/release-notes-1.2.0.rst

```diff
@@ -0,0 +1,482 @@
+.. _ncs_release_notes_120:
+
+|NCS| v1.2.0 Release Notes
+##########################
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products.
+It includes the MCUboot and the Zephyr RTOS open source projects, which are continuously integrated and re-distributed with the SDK.
+
+|NCS| v1.1.0 supports product development with the nRF9160 Cellular IoT device.
+It contains reference applications, sample source code, and libraries for Bluetooth Low Energy devices in the nRF52 Series, though product development on these devices is not currently supported with the |NCS|.
+??? What do we say about nRF5340?
+
+
+Highlights
+**********
+
+* ???
+
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v1.2.0**.
+Check the ``west.yml`` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`Getting the nRF Connect SDK code <cloning_the_repositories_win>` for more information.
+
+
+Supported modem firmware
```

Comment:
Perhaps "Supported modem firmware for Cellular IoT applications" ?

Reply:
A bit too long for the heading, but I added an introduction - ok?

Supported modem firmware
************************

This version of the |NCS| supports the following modem firmware for cellular IoT applications:

* mfw_nrf9160_1.2.0

Reply:
Modem version is not corect. mfw_nrf9160_1.1.1

Reply:
OK, updated.